### DISCUSSION TITLE
Great project! curious discrepancy

### DISCUSSION THREAD

**Discussion by jesuslop** [2Fake/devolo_plc_api#184] [General]
This lib if fantastic. Using it to reboot pesky devolos from a NAS. Strangely invoking from a laptop connected via wifi to a devolo it doesn't work very ok. Nor it does fail always. It works only infrequently (?). That is, the devolo is pingable, web server responds to curl, but the API cannot connect consistently, only by chance. But attacking the same device without passing through wifi (air) links then API works flawless. Maybe there is link-layer peculiarities in the wifi link (broadcast behaviour or maybe UDP quirks). Not a problem for my use case, but wanted to share to other potential users.

**Comment by Shutgun** [comment]
Hi @jesuslop , thanks for your nice words. I forwarded your observation to my friends at devolo. Let's see, if they can find any issue and fix it.
